Switch statement example updated (#37757)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/selecting-from-many-options-with-switch-statements.english.md

## Description
<section id='description'>

> If you have many options to choose from, use a <code>switch</code> statement. A <code>switch</code> statement tests a value and can have many <code>case</code> statements which define various possible values. Statements are executed from the first matched <code>case</code> value until a <code>break</code> is encountered.

Reviewer: ```suggestion
If you have many options to choose from, use a <dfn>switch</dfn> statement. A <code>switch</code> statement tests a value and can have many <dfn>case</dfn> statements which define various possible values. Statements are executed from the first matched <code>case</code> value until a <code>break</code> is encountered.
```
Since this is the first place either of these terms are introduced, it's best to use `dfn`.

> A <code>switch</code> statement can be written as follows:

Reviewer: ```suggestion
Here is an example of a <code>switch</code> statement:
```
This does not define the syntax, exactly, so it's probably best to reinforce that it is just an example.
